sdcm/cluster.py

```diff
@@ -1864,13 +1878,24 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         """.format(mgmt_user=mgmt_user, rsa_id_dst=rsa_id_dst, rsa_id_dst_pub=rsa_id_dst_pub))  # generate ssh public key from private key.
         self.remoter.run('sudo bash -cxe "%s"' % ssh_config_script)
 
+        self.config_scylla_manager_yaml(segments_per_repair=segments_per_repair)
+
         if self.is_docker():
             self.remoter.run('sudo supervisorctl restart scylla-manager')
             res = self.remoter.run('sudo supervisorctl status scylla-manager')
         else:
             self.remoter.run('sudo systemctl restart scylla-manager.service')
             res = self.remoter.run('sudo systemctl status scylla-manager.service')
 
+        if self.is_rhel_like():  # TODO: Add debian and ubuntu support
+            configuring_manager_command = dedent("""
+            scyllamgr_ssl_cert_gen
+            sed -i 's/#tls_cert_file/tls_cert_file/' /etc/scylla-manager/scylla-manager.yaml
+            sed -i 's/#tls_key_file/tls_key_file/' /etc/scylla-manager/scylla-manager.yaml
+            systemctl restart scylla-manager
```

Comment:
Only this command may be different on other distros like deb and ubuntu.
Move the if here and handle the ubuntu and debian as well.
Don't leave the TODO here.

Reply:
@roydahan It's more than this one command, I didn't test the manager on debian yet and I don't think I should merge it without testing

Reply:
@ShlomiBalalis please create a trello task for doing the manager debian/ubuntu support